content/blog/2019/egress-traffic-control-in-istio-part-2/index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Secure Control of Egress Traffic in Istio, part 2
+subtitle: Use Istio Egress Traffic Control to prevent attacks involving egress traffic
+description: Use Istio Egress Traffic Control to prevent attacks involving egress traffic.
+publishdate: 2019-05-29
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,egress,security]
+---
+
+This is part 2 in a new series about secure control of egress traffic in Istio.
+In [the first part in the series](/blog/2019/egress-traffic-control-in-istio-part-1/), I presented the attacks involving
+egress traffic and the requirements we collected for an egress traffic control system.
+In this installment, I describe the Istio way for controlling egress traffic securely and show how Istio prevents the
+attacks.
+
+## Egress traffic control by Istio
+
+To implement secure egress traffic control in Istio, you must
```

Comment:
@rcaballeromx I wrote this post differently - first I tell the readers what they must do, and then I explain them how the steps they took prevent the attacks. It is like to tell the reader:

**The threat:** The attackers want to penetrate your house.
**What the reader should do:** Build a high fence and lock the gate, add a surveillance system.
**How it will prevent the attacks**: 
1. The attacker will try to climb over the fence - they will not be able since it is high. 
1. The attackers will try to open the gate - they will not be able since it is locked.
1. If the attackers will bring a ladder and climb over the fence, the surveillance system will still detect them.

The issue is that for the secure control of egress traffic you apply a set of mechanisms, and together they prevent all possible attacks (that I can think about).